tests/pytorch/test_pytorch_autolog.py

```diff
@@ -1,28 +1,30 @@
 import os
 import pytest
 import pytorch_lightning as pl
+import torch
 from iris import IrisClassification
-from mlflow.pytorch.pytorch_autolog import __MLflowPLCallback
+import mlflow
+import mlflow.pytorch
+from mlflow.pytorch.pytorch_autolog import autolog
 from pytorch_lightning.callbacks.early_stopping import EarlyStopping
 from pytorch_lightning.callbacks import ModelCheckpoint
-from pytorch_lightning.loggers import MLFlowLogger
 
-NUM_EPOCHS = 20
+NUM_EPOCHS = 100
```

Comment:
Reduce the default value for testing to a lower number

Reply:
The earlystopping test cases with patience parameter of 5 was unstable due to lesser number of epochs. I will reduce it to 20 and will set the number of epochs to 40 for the specific test case alone.